src/JustSaying/AwsTools/MessageHandling/SnsMessagePublisher.cs

```diff
@@ -10,12 +10,14 @@
 
 namespace JustSaying.AwsTools.MessageHandling;
 
+#pragma warning disable RS0026 // Do not add multiple overloads with optional parameters
```

Comment:
The analyzers behave differently between Shipped and Unshipped as it can warn you of stuff in Unshipped that would be breaking to change if it had Shipped. I ran into this while working on #1335 after I rebased it onto us adding the analysers.

Reply:
https://github.com/justeattakeaway/JustSaying/pull/1335/commits/366fd642046d9118754410740f27553ef296d7e0

Reply:
Ah, nice, I'll integrate that when I'm next at my computer.

Reply:
Hmm, I can't see a way of changing this to satisfy the analyzer without breaking API compatibility by removing a potentially used default parameter.

I currenlty have all of the .NET 8.0 APIs listed as Unshipped, but should I move them to _shipped_ as the API shape has shipped, it's just a new TFM?

Reply:
Yeah, I think the way to resolve it is to just consider them as shipped, as they're effectively just a copy of existing APIs. That should then make it leave you alone.